src/SignUpSheetAdvance.js

```diff
@@ -1,5 +1,5 @@
-const __SIGNUP_SHEET_HEADER_ROW_NUM = 7;
-const __SIGNUP_SHEET_SECTION_START_COL_NUM = 2;
+const __SIGNUP_SHEET_HEADER_ROW_NUM = SIGNUP_START_ROW;
```

Comment:
Why not just replace these variables rather than aliasing them?

i.e. use `SIGNUP_START_ROW` and `SIGNUP_START_COL` throughout this script

Reply:
I thought it might be easier to leave this as-is, since I'm already having a difficult time figuring out what variables are defined where.

Reply:
Makes sense. I think I had a TODO in one of these files around cleaning up variables, but it's probably worth putting it into an issue to track: #22